### DISCUSSION TITLE
Listener for scroll bar dragging

### DISCUSSION THREAD

**Discussion by hagay3** [katspaugh/wavesurfer.js#3899] [Q&A]
While I'm dragging the scroll bar is there a listener for that case?
In addition, I want to make the scroll bar always be shown and not auto-hide, is it possible to unhide the scroll bar?

**Comment by katspaugh** [comment]
The “scroll” event is probably what you want.
Scrollbar hiding is an OS feature. It cannot be controlled via wavesurfer.
